scripts/bootstrap.sh

```diff
@@ -1,20 +1,59 @@
 #!/usr/bin/env bash

```

Comment:
There is a lot of code repetition, both here and in the Vagrantfile.  We can work on it together if it can be of help.

Reply:
I will be happy to sit on it with you